Other/FarmerJoeOutfit.cs

```diff
@@ -1,43 +1,41 @@
+//cs_include Scripts/CoreAdvanced.cs
 //cs_include Scripts/CoreBots.cs
-//cs_include Scripts/CoreFarms.cs
 //cs_include Scripts/CoreDailies.cs
+//cs_include Scripts/CoreFarms.cs
 //cs_include Scripts/CoreStory.cs
-//cs_include Scripts/CoreAdvanced.cs
-//cs_include Scripts/Hollowborn/HollowbornReapersScythe.cs
+//cs_include Scripts/Dailies/0AllDailies.cs
+//cs_include Scripts/Dailies/LordOfOrder.cs
 //cs_include Scripts/Enhancement/InventoryEnhancer.cs
-//cs_include Scripts/Story/ThroneofDarkness/CoreToD.cs
 //cs_include Scripts/Good/ArchPaladin.cs
-//cs_include Scripts/Story/XansLair.cs
 //cs_include Scripts/Good/BLoD/CoreBLOD.cs
-//cs_include Scripts/Story/Yokai.cs
-//cs_include Scripts/Story/LordsofChaos/Core13LoC.cs
-//cs_include Scripts/Other/Classes/DragonShinobi.cs
 //cs_include Scripts/Good/GearOfAwe/CapeOfAwe.cs
-//cs_include Scripts/Dailies/LordOfOrder.cs
-//cs_include Scripts/Nation/CoreNation.cs
+//cs_include Scripts/Good/GearOfAwe/CoreAwe.cs
+//cs_include Scripts/Good/Paladin.cs
+//cs_include Scripts/Hollowborn/HollowbornReapersScythe.cs
 //cs_include Scripts/Nation/AssistingCragAndBamboozle[Mem].cs
+//cs_include Scripts/Nation/CoreNation.cs
 //cs_include Scripts/Nation/VHL/CoreVHL.cs
-//cs_include Scripts/Dailies/0AllDailies.cs
-//cs_include Scripts/Story/Nation/CitadelRuins.cs
-//cs_include Scripts/Story/QueenofMonsters/Extra/LivingDungeon.cs
-//cs_include Scripts/Story/DragonFableOrigins.cs
-//cs_include Scripts/Story/Glacera.cs
-//cs_include Scripts/Good/GearOfAwe/CoreAwe.cs
-//cs_include Scripts/Other/Weapons/BurningBlade.cs
-//cs_include Scripts/Other/Classes/REP-based/Shaman.cs
+//cs_include Scripts/Other/Classes/DragonShinobi.cs
+//cs_include Scripts/Other/Classes/REP-based/EternalInversionist.cs
 //cs_include Scripts/Other/Classes/REP-based/GlacialBerserker.cs
-//cs_include Scripts/Story/QueenofMonsters/Extra/BrightOak.cs
+//cs_include Scripts/Other/Classes/REP-based/Shaman.cs
 //cs_include Scripts/Other/Classes/REP-based/StoneCrusher.cs
-//cs_include Scripts/Other/Classes/REP-based/EternalInversionist.cs
-//cs_include Scripts/Good/Paladin.cs
-//cs_include Scripts/Other/Weapons/EnchantedVictoryBladeWeapons.cs
-//cs_include Scripts/Story/Tutorial.cs
-//cs_include Scripts/Other/Weapons/DualChainSawKatanas.cs
 //cs_include Scripts/Other/FreeBoostsQuest(10mns).cs
+//cs_include Scripts/Other/Weapons/BurningBlade.cs
+//cs_include Scripts/Other/Weapons/DualChainSawKatanas.cs
+//cs_include Scripts/Other/Weapons/EnchantedVictoryBladeWeapons.cs
+//cs_include Scripts/Story/DragonFableOrigins.cs
+//cs_include Scripts/Story/Glacera.cs
+//cs_include Scripts/Story/LordsofChaos/Core13LoC.cs
 //cs_include Scripts/Story/LordsofChaos/MountDoomSkull.cs
+//cs_include Scripts/Story/Nation/CitadelRuins.cs
+//cs_include Scripts/Story/QueenofMonsters/Extra/BrightOak.cs
+//cs_include Scripts/Story/QueenofMonsters/Extra/LivingDungeon.cs
+//cs_include Scripts/Story/ThroneofDarkness/CoreToD.cs
+//cs_include Scripts/Story/Tutorial.cs
```

Comment:
u cleaned up alotta of stuff in here so i gotta trust u on it can't check every single one of them

Reply:
well know when someone runs it 👍